utils/container.hpp

```diff
@@ -0,0 +1,42 @@
+// Copyright (C) 2018 Bluzelle
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License, version 3,
+// as published by the Free Software Foundation.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+#pragma once
+
+#include <include/bluzelle.hpp>
+#include <bootstrap/bootstrap_peers_base.hpp>
+#include <random>
+
+namespace
+{
+    std::mt19937 gen(std::time(0)); //Standard mersenne_twister_engine seeded with rd()
+}
+
+
+namespace bzn::utils::container
```

Comment:
Is this really going to be used elsewhere?

Reply:
Yeah, I know. But I really wanted to unit test this one, so I just put in utils. My other thought was that it was a good start for a container library.

I'd like to keep it there, but I'm willing to move it to an anonymous namespace. I can live without the unit test now,.